bases/recommended.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "display": "Recommended",
+  
+  "compilerOptions": {
+    /** Strict mode offers the most safety for your application */
+    "strict": true,
+    
+    /** These three options are to improve compatability with the js + babel ecosystem */
```

Comment:
Super minor suggestion - I always found the comments a little distracting when using `--init` and it takes a while to get rid of them.

First world problems though! This looks way more trimmed down than the current auto-generated config 🙌

Reply:
I'm against dropping the comments, you don't edit this file because it lives in a node module

Reply:
It would be nice to have a config file without comments though, since it's a pain to delete them from the config file manually to make it more readable. Perhaps there could be an extra argument alongside `--init` like `--stripComments`...?

Reply:
As mentioned above you can't edit this file, your own tsconfig can have as little or many comments as you'd like

Reply:
My apologies @orta, I thought this was something else